ui/lib/core/addon/templates/components/replication-action-demote.hbs

```diff
@@ -1,47 +1,55 @@
-<h4 class="title is-5 is-marginless">
-  Demote cluster
-</h4>
-<div class="content">
-  {{#if (and
-      (eq replicationMode 'dr')
-      (not model.performance.replicationDisabled)
-    )
-  }}
-
-    <div data-test-demote-warning>
-      <AlertInline
-        @type="danger"
-        @message="Demoting this DR primary cluster
-          would result in a DR secondary and in that mode Vault is read-only. This
-          cluster is also currently operating as a Performance
-          {{capitalize model.performance.modeForUrl}}, demoting it will leave your
-          replication setup without a performance primary cluster until a new
-          cluster is promoted."
-      />
-    </div>
-  {{/if}}
-    <p>
-      Demote this {{replicationDisplayMode}} primary cluster to a {{replicationDisplayMode}} secondary. The resulting secondary cluster will not
-      attempt to connect to a primary, but will maintain knowledge of its cluster
-      ID and can be reconnected to the same set of replication clusters without
-      wiping local storage.
-    </p>
+<div class="action-block is-rounded" data-test-demote-replication>
+  <div class="action-block-info">
+    <h4 class="title is-5 is-marginless">
+      Demote cluster
+    </h4>
     <p>
-      In order to connect the resulting secondary to a new primary, use the <code>Update primary</code> action.
+      Demote this {{if (eq model.replicationMode "dr") "Disaster Recovery" "Performance"}} primary to a secondary.
```

Comment:
Need to update this to simply {{model.replicationModeDisplay}} once [this PR](https://github.com/hashicorp/vault/pull/9120) is updated with the base branch